python/lsst/ts/mtmount/mtmount_commander.py

```diff
@@ -131,6 +137,20 @@ async def do_stopTracking(self, args):
         self.tracking_task.cancel()
         await self.remote.cmd_stopTracking.start(timeout=STD_TIMEOUT)
 
+    # TODO DM-37947: remove this method once we use salobj > 3.7.
```

Comment:
Same question regarding preventing changing ts_xml as above.

Reply:
I am adding this method to ts_salobj's CscCommander. So this version would no longer be needed.